packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2817,6 +2819,18 @@ export class ContainerRuntime
 				forcedFullTree,
 			} as const;
 
+			// Validate that the summary generated by the summarizer nodes is correct.
+			const validateResult = this.summarizerNode.validateSummary();
+			if (!validateResult.success) {
+				const { success, ...loggingProps } = validateResult;
+				const error = new RetriableSummaryError(
+					validateResult.reason,
+					validateResult.retryAfterSeconds,
```

Comment:
Are you intentionally leaving these two in `loggingProps`?  I was expecting you to spread these out on L2825.  `retryAfterSeconds` is a no-op to leave in there (since it's a property of the new error class), but for `reason`, we'll log the same string for both `reason` and `error`.

Up to you, just making sure it's clear what's going on.

Reply:
Yeah, I left it intentionally. I often look at message and reason in telemetry and it will be nice to have them both say why it failed.